content/quickstart/aws/tutorial-rest-api.md

```diff
@@ -29,16 +34,16 @@ pulumi new hello-aws-javascript
 
 You can accept the defaults for this command. For instance, you can change the AWS region to `us-west-2`.
 
-![Run Pulumi new](/images/quickstart/hello/Quickstart1.png){:width="700px"}
+<img src="/images/quickstart/hello/Quickstart1.png" alt="Run Pulumi new" width="700">
```

Comment:
Hugo doesn't support the `{:...}` syntax that Jekyll does for specifying widths and adding classes, so just using an `<img>` tag for these instead.